### ISSUE TITLE
Create logo

### ISSUE THREAD

**Issue by frewsxcv** [frewsxcv/afl.rs#66]
Might fun to do something like [this](https://groups.google.com/forum/#!topic/afl-users/_kNrt_n5Nug) but with the rust logo instead of a rabbit

http://lcamtuf.coredump.cx/afl/rabbit.gif

https://groups.google.com/d/msg/afl-users/_kNrt_n5Nug/vcZvl2Y7C7EJ

**Comment by Nemo157** [comment]
This sounded interesting so I started writing a little project to generate it https://github.com/Nemo157/afl-rs-logo. Needs the filtering and sorting stages automated still, but by manually choosing a few interesting images from the corpus I got https://github.com/Nemo157/afl-rs-logo/blob/master/generated/wip1.gif

**Comment by frewsxcv** [comment]
Wow, this is amazing work @Nemo157, thanks for doing this! I'll take a closer look over the next couple days at the code to see how you accomplished this.

**Comment by frewsxcv** [comment]
@Nemo157 I checked out your repo, followed your (very clear) instructions, and after a couple hours of fuzzing, I was able to generate a very cool gif :) Even though I was able to generate a gif, I still preferred your gifs you created above. I chose one of them and added it into the README in [this pull request](https://github.com/frewsxcv/afl.rs/pull/106). If you want to be attributed somewhere in the README for your work on the logo, let me know (though I made the link on the logo.gif in the README link to this GitHub issue).

Thanks again for the work you did here, I really appreciate the effort 🎊 😍 🎉
